packages/spec-api/test/expectation.test.ts

```diff
@@ -0,0 +1,61 @@
+import { describe, expect, it } from "vitest";
+import { RequestExpectation } from "../src/expectation.js";
+import { RequestExt } from "../src/types.js";
+
+describe("expectation test suite", () => {
```

Comment:
can you remove those top level describe. They are redundant with the filename as vitest which already group per file